### ISSUE TITLE
Localization support?

### ISSUE THREAD

**Issue by non-descriptive** [vito/booklit#18]
Is there any `i18n` or `l10n` support already?

**Comment by vito** [comment]
Not at the moment, sorry. Where do you think this kind of thing should begin? Plugin function names? Error messages? :thinking:

**Comment by non-descriptive** [comment]
No, i meant generated site, not an booklit itself. Something like make some extra folders with translated string and some switch button in output html. Or maybe just like in wikipedia with some language picks at the sidebar.  That kind of localization.

**Comment by vito** [comment]
Ah, I see. I haven't given any thought to it to be honest. Would you be maintaining translations for each `.lit`  document? Maybe they could use [`\styled`](https://vito.github.io/booklit/baselit.html#styled) to use different templates or [`\set-partial`](https://vito.github.io/booklit/baselit.html#set-partial) to parameterize them?

**Comment by non-descriptive** [comment]
Not really tried any of booklit functionality yet. Just checking better documentation-like generators: mdbook, gitbook, doxygen and booklit on the way. Struggle with multilingual support. I'll try things you adviced. Thank you. 
Also is there gonna be any PDF renderer? Demo site looks like almost fit to book format.

**Comment by vito** [comment]
@non-descriptive I don't have time to work on a PDF renderer at the moment but it definitely fits in the spirit of the design. 🙂 [Scribble](https://docs.racket-lang.org/scribble/), which heavily inspired Booklit, supports rendering to PDF for example.
